web/app.js

```diff
@@ -362,6 +362,7 @@ const PDFViewerApplication = {
     if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("TESTING")) {
       Object.assign(opts, {
         docBaseUrl: x => x,
+        capCanvasAreaFactor: x => parseInt(x),
```

Comment:
Nit: Alphabetical order please.
```suggestion
        capCanvasAreaFactor: x => parseInt(x),
        docBaseUrl: x => x,
```